src/managers/GitConfigManager.js

```diff
@@ -2,17 +2,24 @@ import { GitConfig } from '../models/GitConfig.js'
 
 export class GitConfigManager {
   static async get({ fs, gitdir }) {
+    if (this.config) {
+      return this.config
+    }
+
     // We can improve efficiency later if needed.
     // TODO: read from full list of git config files
-    const text = await fs.read(`${gitdir}/config`, { encoding: 'utf8' })
-    return GitConfig.from(text)
+    this.config = GitConfig.from(
+      await fs.read(`${gitdir}/config`, { encoding: 'utf8' })
+    )
+    return this.config
   }
 
   static async save({ fs, gitdir, config }) {
     // We can improve efficiency later if needed.
     // TODO: handle saving to the correct global/user/repo location
+    this.config = config
     await fs.write(`${gitdir}/config`, config.toString(), {
-      encoding: 'utf8',
```

Comment:
Please don't change the style the library is using.

Reply:
Sorry about that, I am not sure what happened there I must have something misconfigured.
I have run the formatting script now.